bin/problem.py

```diff
@@ -733,12 +740,13 @@ def validate_data(problem, mode: validate.Mode, constraints: dict | bool | None
                 testcases = problem.testcases(mode=mode)
             case validate.Mode.ANSWER:
                 assert not problem.interactive
+                assert not problem.multipass
                 problem.validators(validate.AnswerValidator, check_constraints=check_constraints)
                 problem.validators(validate.OutputValidator, check_constraints=check_constraints)
                 testcases = problem.testcases(mode=mode)
             case validate.Mode.INVALID:
                 problem.validators(validate.InputValidator)
-                if not problem.interactive:
+                if not problem.interactive and not problem.multipass:
```

Comment:
I see a lot of `problem.interactive or problem.multipass`. Should we introduce `problem.simple` or so?
(Really not quite sure though; it's hard to find a good name for it and clarity may be lost. Keeping as it is is also fine.)

Reply:
i think both makes clearer what's going on... for now. And i am not sure if i have to undo some places later ^^